src/core/Env.js

```diff
@@ -142,6 +142,10 @@ getJasmineRequireObj().Env = function(j$) {
       if (configuration.hasOwnProperty('hideDisabled')) {
         config.hideDisabled = configuration.hideDisabled;
       }
+
```

Comment:
Per above, please remove since we're not actually making this configurable

Reply:
done